pkg/yurtctl/util/kubernetes/util.go

```diff
@@ -608,12 +614,36 @@ func GetOrCreateJoinTokenString(cliSet *kubernetes.Clientset) (string, error) {
 	}
 
 	klog.V(1).Infoln("[token] creating token")
-	if err := tokenphase.CreateNewTokens(cliSet, []kubeadmapi.BootstrapToken{{Token: token, Usages: kubeadmcontants.DefaultTokenUsages, Groups: kubeadmcontants.DefaultTokenGroups}}); err != nil {
+	if err := tokenphase.CreateNewTokens(cliSet,
+		[]kubeadmapi.BootstrapToken{{
+			Token:  token,
+			Usages: kubeadmcontants.DefaultTokenUsages,
+			Groups: kubeadmcontants.DefaultTokenGroups,
+		}}); err != nil {
 		return "", err
 	}
 	return tokenStr, nil
 }
 
+// isBootstrapToken checks if the token can be used as a bootstrap token.
+func isBootstrapToken(token *kubeadmapi.BootstrapToken) bool {
```

Comment:
isBootstrapToken is used to check token has the right usages and groups, so how about define the func name to `usagesAndGroupsAreValid`?

Reply:
@rambohe-ch thanks for the comment, I agree that`usagesAndGroupsAreValid` is better, updated.